portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,190 @@
+@import url('https://fonts.googleapis.com/css?family=Major Mono Display');
+@import url('https://fonts.googleapis.com/css?family=Codystar');
+@import url('https://fonts.googleapis.com/css?family=Nothing You Could Do');
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin-left: 15%;
+  margin-right: 15%;
 }
 
 #greeting-container {
   margin-top: 20px;
+  
+}
+
+#TechCorner {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+#Contact {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+
+img.image {
+    float: right;
+    padding: 20px;
```

Comment:
Generally padding is for spacing inside something and margin is for spacing between elements. The behavior tends to be similar but the semantics is different

Reply:
I didn't really change this portion of the code but somehow it was working in the previous deployed website but now it is:)